### ISSUE TITLE
Resources Not Found

### ISSUE THREAD

**Issue by nick-thompson** [cwilso/WebMIDIAPIShim#2]
Hey Chris,

I get these two 404's from my local dev server when I try loading up the WebMIDIAPIShim:

localhost - - [29/Oct/2012 21:04:33] "GET /lib/net/abumarkub/midi/MIDIApplet.class HTTP/1.1" 404 -
localhost - - [29/Oct/2012 21:04:33] "GET /lib/net/abumarkub/midi/MIDIApplet/class.class HTTP/1.1" 404 -

Those files don't appear to be part of this repo, or your fork of MIDIBridge, and I'm not sure where else i might look to find them, but it's breaking the shim on my machine :(

**Comment by cwilso** [comment]
Hey Nick - what browser and OS are you running?

**Comment by nick-thompson** [comment]
Ah, sorry, should've put that in my first post. That's running the latest stable release of Chrome on OS X 10.6.8.

**Comment by cwilso** [comment]
OK, that's just odd.

That shouldn't be requested like that; that's the name of the MIDIBridge
Java class.  This is from a straight copy of the WebMIDIAPIShim?  (I.e. you
didn't modify it at all?)  Do you already have Java enabled on the browser?
 (chrome://plugins/ should show Java in the list, as enabled.)

Can you open up the dev tools (option-command-I), go to "Elements", look
for "<div id='midibridge-java'>", open it up and check to make sure it
contains

<applet id="midibridge-applet" code="net.abumarkub.midi.MIDIApplet.class"
archive="midiapplet-0.6.3.jar" codebase="lib" width="1" height="1"
mayscript=""><param name="minJavaVersion" value="1.6"></applet>

Java is picky about the 'lib' directory... might be related to that.

-C

**Comment by nick-thompson** [comment]
Hm.. So I found this error using the equipment in a computer lab on campus (I'm a student, and my midi controllers aren't currently with me at school), and when I returned to that lab to inspect the error after your last comment, the shim loaded perfectly. I have no idea what may have changed, except that I'm sure the computer I was using was restarted in between uses. Both cases were an exact copy of the WebMIDIAPIShim (no modifications), and I also tried with an exact copy of your midi-synth project. (Both projects encountered this problem yesterday, and neither of them today).

So either something changed on that computer in between trials, or I was doing something wrong the first time. Either way it's probably safe to assume your project is ok. I'll close this thread, and if I see any similar issues in the future I'll make sure to let you know. Sorry to bother you!
